src/Trixi.jl

```diff
@@ -53,7 +59,23 @@ using FillArrays: Ones, Zeros
 using ForwardDiff: ForwardDiff
 using HDF5: HDF5, h5open, attributes, create_dataset, datatype, dataspace
 using LinearMaps: LinearMap
-using LoopVectorization: LoopVectorization, @turbo, indices
+if _PREFERENCE_LV
+    using LoopVectorization: @turbo, indices
+else
+    using LoopVectorization: indices
+    macro turbo(exprs...)
+        body = nothing
+        for expr in exprs
+            if expr.head == :for
+                body = expr
+            end
+        end
+        @assert body !== nothing
+        # esc(Base.SimdLoop.compile(body, Symbol("julia.ivdep")))
+        return esc(body)
+    end
```

Comment:
Does this work with nested `for` loops etc.?

Reply:
Yes, my original plan was to basically turn this into `@simd`, but that wouldn't work with nested for loops

Reply:
👍